eng/Packages.Data.props

```diff
@@ -484,8 +492,13 @@
     <PackageDownload Include="Azure.Sdk.Tools.Testproxy" Version="[$(TestProxyVersion)]" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(IsClientLibrary)' == 'true' and $(MSBuildProjectName.StartsWith('Microsoft.ClientModel.TestFramework'))">
+  <ItemGroup Condition="'$(IsClientLibrary)' == 'true' and (
+      $(MSBuildProjectName.StartsWith('Microsoft.ClientModel.TestFramework'))
+      or $(MSBuildProjectName.StartsWith('Azure.AI.Projects.OpenAI.Tests'))
+      or $(MSBuildProjectName.StartsWith('Azure.AI.Projects.Agents.Tests'))
+    )">
     <PackageReference Update="NUnit" Version="4.4.0" />
+    <PackageReference Update="NUnit3TestAdapter" Version="5.1.0" />
```

Comment:
This is another one that I'd hold off on.   A repository-wide upgrade of packages will follow the BCL 10.x updates.   I'd rather we include it there unless there's a specific need for it here.